hs_err_pid29889.log

```diff
@@ -0,0 +1,1043 @@
+#
```

Comment:
Seems it is part of those files that need to be caught by gitignore and it is strange that it is still here. 
Or I missed something, or it should be deleted, what do you think?

Reply:
Thanks for noticing it. Removed :)